File: Uri Online Judge (beecrowd)/1134 - Type of Fuel.py
# -*- coding: utf-8 -*-

"""
Escreva a sua solução aqui
Code your solution here
Escriba su solución aquí
"""

n = int(input())
alcool = 0
gasolina = 0
diesel = 0
while n != 4:
    if n == 1:
        alcool = alcool + 1
    elif n == 2:
        gasolina = gasolina + 1
    elif n == 3:
        diesel = diesel + 1
    elif n == 4:
        break
    n = int(input())

print("MUITO OBRIGADO")
print("Alcool:", alcool)
print("Gasolina:", gasolina)
print("Diesel:", diesel)

File: Uri Online Judge - beecrowd/1134 - Type of Fuel.cpp
// type of fuel
#include <iostream>

using namespace std;

int main()
{
    int c, a = 0, g = 0, d = 0;

    cin >> c;

    while (c != 4)
    {
        switch (c)
        {
        case 1:
            a++;
            break;
        case 2:
            g++;
            break;
        case 3:
            d++;
            break;
        case 4:
            break;
        }
        cin >> c;
    }

    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << a << endl;
    cout << "Gasolina: " << g << endl;
    cout << "Diesel: " << d << endl;

    return 0;
}
